### DISCUSSION TITLE
ProSearch and Consent logging: Disable IP address storage/logging

### DISCUSSION THREAD

**Discussion by creativearc-paul** [ExpressionEngine/ExpressionEngine#4579] [Feature Requests]
### Suggest an idea for ExpressionEngine

We have a health care client, for whom storing any identifiable data is a legal concern.

It'd be great to have a setting so both ProSearch and Consent Logging (and perhaps other core EE logging) could be set to refrain from storing IP addresses.

I didn't see settings for this, so apologies if I'm overlooking something.

<img width="1252" alt="image" src="https://github.com/user-attachments/assets/37aa1178-0eda-4212-81f8-63dcc29ac4de">

<img width="1169" alt="image" src="https://github.com/user-attachments/assets/29f68d51-02b1-4fdd-a33a-56715030e9cf">

**Comment by intoeetive** [comment]
We do have a setting for that under Settings -> Logging.

Or you can set `$config['anonymize_consent_logs'] = 'ip_address';` in your config file
